docs/en/observability/aws-deploy-elastic-serverless-forwarder.asciidoc

```diff
@@ -160,7 +204,9 @@ A list of outputs (i.e. forwarding targets) for the Elastic Serverless Forwarder
 
 `inputs.[].outputs.[].type`:
 
-The type of the forwarding target output (currently only `elasticsearch` supported).
+The type of the forwarding target output (currently only `elasticsearch` and `logstash` are supported).
+
+NOTE: `logstash` target output is currently offered as Technology Preview.**
```

Comment:
```suggestion
NOTE: The `logstash` target output is currently offered as a **Technology Preview.**
```
Editing in case you keep this. Looks like you wanted this to be bold?